src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts

```diff
@@ -307,4 +309,17 @@ export class DBiamPersonenkontextRepo {
 
         return groupedByPerson;
     }
+
+    public async getPersonenkontextRollenCount(): Promise<RollenCount[]> {
+        const query: string = `SELECT rollenart, COUNT(*) as count
+                               FROM (
+                                   SELECT DISTINCT person_id, rollenart
+                                   FROM personenkontext
+                                   JOIN rolle ON personenkontext.rolle_id = rolle.id
+                               ) as distinct_roles
+                               GROUP BY rollenart;`;
+
+        const result: RollenCount[] = await this.em.execute(query, []);
+        return result;
+    }
```

Comment:
Would be nicer to use the MikroORM query builder for safety, but I'll accept this for now

Reply:
fair, i can do it for the next release